internal/command/import.go

```diff
@@ -164,9 +164,7 @@ func (c *ImportCommand) Run(args []string) int {
 	}
 
 	// Load the backend
-	b, backendDiags := c.Backend(&BackendOpts{
-		BackendConfig: config.Module.Backend,
-	})
+	b, backendDiags := c.Meta.prepareBackend(config.Root.Module)
```

Comment:
I'm a bit confused about the `config.Module` -> `config.Root.Module` change here - what is the difference and rationale?